components/popover/metadata/popover.yml

```diff
@@ -1,90 +1,606 @@
 name: Popover
 SpectrumSiteSlug: https://spectrum.adobe.com/page/popovers/
+sections:
+- name: Migration Guide
+  description: |
+    ### New Popover Positions
+    - Popover has 22 available positions. 10 positions use logical properties.
+    - Popover position and distance to source should be handled by implementation.
+    - In this document, Popover positions only impact default tip positioning.
+    - Default tip position is centered on the edge for top/bottom/left/right/start/end positions.
+    - Default tip position is distance from edge == popover corner radius for all other positions.
```

Comment:
Does "==" mean "to"?

Reply:
Old school "is equal to" but I can see that this is not a clear way to communicate here. Updated to use words. Thanks!!!